src/pages/BillableMetricsList.tsx

```diff
@@ -35,7 +35,12 @@ gql`
 const BillableMetricsList = () => {
   const { translate } = useInternationalization()
   let navigate = useNavigate()
-  const { data, error, loading, fetchMore } = useBillableMetricsQuery({
+  const {
+    data,
+    error,
+    loading,
+    fetchMore: fetchMoreWallets,
```

Comment:
Why renaming it ? You just have one fetchMore ?

Reply:
Also it does not fetchMore Wallets here... it fetches more BillableMetrics...

Reply:
Arf did that on the wrong file 😓 Also, this change in a different file would make sense only on a following PR where we'll have 2 fetch (one for wallet, other for walletTransactions)

will simply discard this change here sorry for that. Good catch